### ISSUE TITLE
incompatible redefinition of 'Rectangle'

### ISSUE THREAD

**Issue by jcweaver997** [irishgreencitrus/raylib.v#1]
Hello,
I'm using the latest version of V `V 0.2.4 ddc62ab` and the trying to run the example program in the README.
The full output I'm getting is 
```
PS C:\Users\jc\Documents\git\vgame> v -cg run vgame.v
C:/Users/jc/AppData/Local/Temp/v/vgame.2868074894925457026.tmp.c:432: warning: WINVER redefined
In file included from C:/Users/jc/AppData/Local/Temp/v/vgame.2868074894925457026.tmp.c:682:
C:/Users/jc/.vmodules/irishgreencitrus/raylibv/include/raylib.h:226: error: incompatible redefinition of 'Rectangle'
builder error: 
==================
C error. This should never happen.

This is a compiler bug, please report it using `v bug file.v`.

https://github.com/vlang/v/issues/new/choose

You can also use #help on Discord: https://discord.gg/vlang
```

**Comment by irishgreencitrus** [comment]
yeah, i'm aware of this issue although i'm not sure what causes it just yet. thanks for the report, this is now top of my TODO list!

**Comment by irishgreencitrus** [comment]
The error is caused by conflicts with the WINAPI. I'll see if i can fix it

**Comment by irishgreencitrus** [comment]
AFAIK this is what's happening: raylib defines a class known as Rectangle, and so does the WINAPI. not sure how this will fix atm, although it's not a specific issue with raylib.v. Temporary fix could be prefixing all of raylib's c calls with `raylib_` or something like that. this wouldn't change V's calls so let me know if I should do that

**Comment by jcweaver997** [comment]
Seems like it's more than Rectangle; its also CloseWindow, ShowCursor, LoadImage, and DrawText to name a few. I agree, prefixing everything is the best fix for now. Thanks for looking into this :)

**Comment by Larpon** [comment]
Somewhat related I think: https://github.com/raysan5/raylib/issues/857

**Comment by spytheman** [comment]
Also <https://github.com/raysan5/raylib/issues/1217>

**Comment by irishgreencitrus** [comment]
Fixed in latest commit! :confetti_ball:

**Comment by kudorgyozo** [comment]
Was this solved? I'm getting this error with raylib right now.
